### ISSUE TITLE
idea-ultimate@2022.1.1-221.5591.52: decompress error

### ISSUE THREAD

**Issue by wjq98** [ScoopInstaller/Extras#8520]
#### Scoop Configuration

```json
//# Your configuration here
```

**Comment by issaclin32** [comment]
It works fine on my side.

I assume the problem is caused by some files being blocked by **antivius** softwares.

```
> scoop install idea-ultimate
Installing 'idea-ultimate' (2022.1.1-221.5591.52) [64bit]
Loading ideaIU-2022.1.1.win.zip from cache.
Checking hash of ideaIU-2022.1.1.win.zip ... ok.
Extracting ideaIU-2022.1.1.win.zip ... done.
Running installer script...
INFO  Copying idea.properties file...
Linking D:\scoop\apps\idea-ultimate\current => D:\scoop\apps\idea-ultimate\2022.1.1-221.5591.52
Creating shim for 'idea'.
Creating shortcut for JetBrains\IDEA Ultimate (idea64.exe)
Persisting IDE\bin\idea.properties
Persisting IDE\bin\idea.exe.vmoptions
Persisting IDE\bin\idea64.exe.vmoptions
Persisting profile
'idea-ultimate' (2022.1.1-221.5591.52) was installed successfully!
```

**Comment by issaclin32** [comment]
I am closing the issue because the issue has not been active for a while.
Feel free to leave comment or open a new issue if problem still exists.
